test/param_build_test.c

```diff
@@ -184,6 +260,8 @@ static int template_private_test(int tstid)
         || !TEST_true(OSSL_PARAM_BLD_push_uint32(bld, "i32", 1532))
         || !TEST_true(OSSL_PARAM_BLD_push_uint64(bld, "i64", 9999999))
         || !TEST_true(OSSL_PARAM_BLD_push_size_t(bld, "st", 65537))
+        || !TEST_ptr(zbn = BN_secure_new())
```

Comment:
You need to free zbn and zbn_res - CI is relevant.

Reply:
Fixed